shacl12-test-suite/tests/core/property/memberShape-001.ttl

```diff
@@ -0,0 +1,97 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix ex: <http://datashapes.org/sh/tests/core/property/memberShape-001.test#> .
+@prefix mf: <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix sht: <http://www.w3.org/ns/shacl-test#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ex:Agenda
+  rdf:type rdfs:Class ;
+  rdf:type sh:NodeShape ;
+  rdfs:label "Agenda" ;
+  sh:property ex:Agenda-speakerOrder ;
+.
+
+ex:Agenda-speakerOrder
+  sh:path ex:speakerOrder ;
+  sh:memberShape [
+      a sh:NodeShape ;
+      sh:nodeKind sh:IRI ;
+    ] ;
+  sh:maxCount 1 ;
+.
+
+ex:Alice
+  rdf:type ex:Person ;
+  rdfs:label "Alice" ;
+.
+
+ex:Bob
+  rdf:type ex:Person ;
+  rdfs:label "Bob" ;
+.
+
+ex:Charlie
+  rdf:type ex:Person ;
+  rdfs:label "Charlie" ;
+.
+
+ex:Person
+  rdf:type rdfs:Class ;
+  rdf:type sh:NodeShape ;
+  rdfs:label "Person" ;
+.
+
+# Valid agenda with all IRIs in the list
+ex:agenda-1
+  rdf:type ex:Agenda ;
+  ex:speakerOrder (
+      ex:Alice
+      ex:Bob
+      ex:Charlie
+    ) ;
+  rdfs:label "Agenda 1" ;
+.
+
+# Invalid agenda with a blank node in the list
+ex:agenda-2
+  rdf:type ex:Agenda ;
+  ex:speakerOrder (
+      ex:Alice
+      ex:Bob
+      [ rdfs:label "Charlie" ]
+    ) ;
+  rdfs:label "Agenda 2" ;
+.
+
+<>
+  rdf:type mf:Manifest ;
+  mf:entries (
+      <memberShape-001>
+    ) ;
+.
+
+<memberShape-001>
+  rdf:type sht:Validate ;
+  rdfs:label "Test of sh:memberShape at property shape 001" ;
+  mf:action [
+      sht:dataGraph <> ;
+      sht:shapesGraph <> ;
+    ] ;
+  mf:result [
+      rdf:type sh:ValidationReport ;
+      sh:conforms "false"^^xsd:boolean ;
+      sh:result [
```

Comment:
The blank Charlie node might need to be given a `_:b0` style name if it's supposed to be linked as `sh:value`.